### DISCUSSION TITLE
Welcome to PlantIT!

### DISCUSSION THREAD

**Discussion by wpbonelli** [Computational-Plant-Science/plantit#63] [General]
Feel free to use this thread (or start another one) to:
  * share feedback
  * request features
  * describe your use case

If you have a bug to report, please [file an issue](https://github.com/Computational-Plant-Science/plantit/issues/new).

**Comment by dschneiderch** [comment]
Hi, when I try to login with my cyverse account I get this page:
![image](https://user-images.githubusercontent.com/7461221/117729378-999c3600-b19f-11eb-804e-65f134f2ed24.png)

**Reply by wpbonelli** [reply]
Hi @dschneiderch, thanks for the bug report! I have pushed a fix. Please let me know if the problem persists but I believe it should be solved.

**Reply by dschneiderch** [reply]
Hi, I’m still having a problem. Logging into DE on cyverse works fine.

![image](https://user-images.githubusercontent.com/7461221/117891605-0b41b600-b26c-11eb-9fd6-60bc716c9ced.png)

**Reply by wpbonelli** [reply]
Alas, JWTs from CyVerse can be much larger than anticipated. A fix for this just went out. Thanks so much for your patience, this is a great help stabilizing the platform
